### DISCUSSION TITLE
ZW112 AEON Labs / Door/Window Sensor 6

### DISCUSSION THREAD

**Discussion by tonymartino** [zwave-js/zwave-js#5023] [Q&A] [answered]
Hello!
I picked up the Zooz ZST10 700 stick since I have a 700 series door lock arriving soon. Since I wanted to move my Home Assistant installation from an RPI 4 back to one of my servers (Home Assistant Operating System in a VM), I excluded all of my Z-Wave devices from the RPI (and the Aeotec Z-Stick Gen 5) and started adding them to the new installation. When it came to the ZW112, I was surprised to find no enabled sensor entities:
![image](https://user-images.githubusercontent.com/65514761/188739407-fbdf7a65-7729-4a45-a18d-021a100819bd.png)
No biggy, I enabled both **sensor.front_door_sensor_access_control_door_state** and **binary_sensor.front_door_sensor_sensor_state_any** and found that, when I open or close the door, the state of the access control door state sensor is updated, great. However, the values are... strange:
![image](https://user-images.githubusercontent.com/65514761/188740822-bb8f900b-fc74-497a-a6e3-407f66cdc4b5.png)
Closing the door is great but opening it changes the state to **22**. I tried re-interviewing the node but that didn't help. I don't mind using **22** as a trigger but I'm concerned that something is wrong. Has anybody ran into this kind of problem before?

Thank you!

**Comment by tonymartino** [comment, marked as answer]
Just experienced the same thing with a ZD2105 by Vision Security. On the other hand, a WADWAZ-1 door sensor just turns its **Sensor state (Any)** state to on or off.

**Reply by AlCalzone** [reply]
22 means open. Please share the node dump you can download in the dropdown under device info. There might be something missing.

**Reply by tonymartino** [reply]
Here it is!
[zwave_js-1016f773b06ff92fefc9bc970b9b11f1-Door_Window Sensor 6-e4a53a735bb7d00320f8f4f17828d227.json.txt](https://github.com/zwave-js/node-zwave-js/files/9501165/zwave_js-1016f773b06ff92fefc9bc970b9b11f1-Door_Window.Sensor.6-e4a53a735bb7d00320f8f4f17828d227.json.txt)

There's only one state listed there. Interesting. Where does this data come from?
Edit: I've been playing with this and I see that it's the current state.

**Reply by AlCalzone** [reply]
This data should normally get populated during the interview. Maybe something went wrong there.
Can you start a log (on level debug!) like mentioned here: https://www.home-assistant.io/integrations/zwave_js#how-to-access-the-z-wave-logs
then re-interview your device (in the same menu as before)?

Once the interview is done, please download the log and drag it here to attach it.

**Reply by tonymartino** [reply]
Hi Al,
Here is the file:
[zwave_js.log](https://github.com/zwave-js/node-zwave-js/files/9507652/zwave_js.log)

One thing I noticed when re-interviewing: I woke the device up and it got to the Protocol Info. stage then the device went to sleep again. I had to wake it up once more in order for the interview to complete. Not sure if it's the device that decided to go to sleep or if Z-Wave JS told it to but thought I'd mentioned just in case it's an issue you're able to deal with.

Thanks!

**Reply by AlCalzone** [reply]
Looks like zwave-js [incorrectly](https://github.com/zwave-js/node-zwave-js/issues/5026) assumes it to be asleep after ProtocolInfo. And the door state labels are also a bug: https://github.com/zwave-js/node-zwave-js/issues/5027

**Reply by tonymartino** [reply]
Al, thanks so much for your attention to this! I'm happy to help with more logs, testing, etc.

-Tony
